README.md

```diff
@@ -0,0 +1,88 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Rails Back End
+
+> Understand the setup process for adding React/Redux to rails application.
+
+This project is made as part of Microverse program.
+
+you can view the front-end of this app [here](https://github.com/oluyaratosin123/Hello-React-Front-End)
```

Comment:
- [ ] For more detailed documentation, kindly replace this link with the more appropriate link as contained in your PR summary